fleche/doc.ml

```diff
@@ -221,7 +218,7 @@ let state_recovery_heuristic doc st v =
   | _ -> st
 
 let debug_parsed_sentence ~ast =
-  let loc = Coq.Ast.loc ast |> Option.get in
+  let loc = Coq.Ast.loc ast |> Option.default (Loc.make_loc (0, 0)) in
```

Comment:
This is incorrect.

Reply:
What do you mean? If `Coq.Ast.loc` returns `None` loc, what do we do?

Reply:
As all the other changes, if the location is not there an error must be thrown.

I know this is not perfect but let's keep the Option.get for now

Reply:
It will never return `None`, the typing upstream is too weak due to the ast type being used for both parsing and term synthesis.